conf/application.conf

```diff
@@ -38,22 +38,6 @@ play.http.errorHandler = "uk.gov.hmrc.helpfrontend.config.ErrorHandler"
 # In server environments the secret comes from app-config-common
 # play.http.secret.key="rQCoRUXyQl0i1HfzligLGlpyCc0LmcfrZYPjxQmO7iCz0Rr3ulesXmkFqFlujuJk"
 
-microservice {
-  metrics {
-    graphite {
-      enabled = true
-    }
-  }
-}
-
-metrics {
-  enabled = true
-}
-
-auditing {
-  enabled = true
-}
-
```

Comment:
these are all set to false by bootstrap - and true in app-config-common, removing them means we don't try to send stuff when running tests (which generates errors in console)